### PR TITLE
Styleguide

### PR DESCRIPTION
**Pull request by jimhunty**
We have talked about adding a Styleguide to the components to capture colours, vertical rhythm, fonts etc.

For the Article Label story, we need a one to one match with sections and their colours, as section comes out of TPA.

We can then use this package in an upcoming story.

N.B. We do not have all the reference names for the colours. Waiting on those who know to come back however, this shoudnt block this PR.

### CODE REVIEW

File: packages/styleguide/colours/section.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const sectionColours = {
+  artsfeatures: "#622956",
+  business: "#005b8d",
+  bricksmortar: "#006469",
+  comment: "#850029",
+  Comment: "#850029",
```
Comment (aronshamash) [inline packages/styleguide/colours/section.js]: Can you explain this ?

Comment (colincclark) [reply packages/styleguide/colours/section.js]: @aronshamash https://github.com/newsuk/times-public-api/blob/167387326b64f3ec3397983f49529f0ce33086df/src/schemas/article.graphql#L46

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/7420eee807143ae997c1eae083be181a59d73940/dextrose-test/index.html

**Comment by jimhunty** [comment]
Thanks @aronshamash for the update. I will raise a PR on times-public-api to add these to the flow types.

File: packages/styleguide/colours/section.js (hunk 1 above)
Comment (killanaca) [reply packages/styleguide/colours/section.js]: and just for ref, this is the mapping via news-desk https://github.com/newsuk/times-public-api/blob/master/src/utils/news-desk-mapping.js

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/297c9600662910089be4ac7bf5c123d4740da12d/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/f79f564e188ccfb2f783b2607ea5756bd2a3ce20/dextrose-test/index.html

**Pull request merged**